kum-archaeodb-web/src/app/categories/category-list/category-list.component.ts

```diff
@@ -36,8 +36,13 @@ export class CategoryListComponent implements OnInit {
       if (category) {
         console.log("Add category: \n" + JSON.stringify(category, null, 2));
         this.service.addCategory(category).subscribe(data => {
-          this.getCategories();
-        })
+            this.getCategories()
+          },
+          error => {
+            if (error.status == 400) {
+              alert('Категорія з таким ім\'ям вже існує')
```

Comment:
Let's use the [MatSnackBar](https://material.angular.io/components/snack-bar/overview) for error notifications, with auto-hide after a few seconds.
Browser alerts may be a little annoying.

Reply:
fixed in 14e5e2d07